bundles/binding/org.openhab.binding.caldav-command/src/main/java/org/openhab/binding/caldav_command/internal/CalDavBinding.java

```diff
@@ -0,0 +1,509 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.caldav_command.internal;
+
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Dictionary;
+import java.util.List;
+import java.util.Map.Entry;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.openhab.binding.caldav_command.CalDavBindingProvider;
+
+import org.openhab.core.binding.AbstractBinding;
+import org.openhab.core.items.Item;
+import org.openhab.core.items.ItemNotFoundException;
+import org.openhab.core.items.ItemRegistry;
+import org.openhab.core.library.types.DateTimeType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.openhab.core.types.TypeParser;
+import org.openhab.io.caldav.CalDavEvent;
+import org.openhab.io.caldav.CalDavLoader;
+import org.openhab.io.caldav.EventNotifier;
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.cm.ManagedService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+	
+
+/**
+ * This is the default implementation of the caldav binding
```

Comment:
what do you mean by "default"? Will there be other? Please also enhance this documentation and briefly explain the purpose/functionality of this class